src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqUserRepository.kt

```diff
@@ -86,12 +108,19 @@ class JooqUserRepository : JooqRepository(), UserRepository
 
         // set scopeId to null if USER_ROLE.SCOPE_ID is an empty string,
         // so that scopeId and scopePrefix are consistently null/not null
-        scopeId = if (scopeId.isEmpty()) { null } else { scopeId }
+        scopeId = if (scopeId.isEmpty())
+        {
+            null
+        }
+        else
+        {
+            scopeId
+        }
 
         return RoleAssignment(
                 record[ROLE.NAME],
-                scopeId,
-                record[ROLE.SCOPE_PREFIX])
+                record[ROLE.SCOPE_PREFIX],
+                scopeId)
     }
```

Comment:
note, just changed this constructor around so that prefix comes before id, as is the case everywhere else